changelog/1.19.0.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.19.0.
+
+### Features ✨
+
+- web: Resource quotas have been introduced to Organizations. Resource Quotas define
+  maximum resource utilization applied to each member within the Organization they're
+  applied to. Modify them **Manage > Organizations > Edit Organization**.
+- web: Add site and organization workspace as code templates
+- web: An administrative control over Dev URL access levels was added.
+- web: By default, the CVM option for creating custom environments is now checked
```

Comment:
Needs documentation

Reply:
Does this mean that CVMs are enabled at the admin level (Admin > Infrastructure> Workspace Container Runtime --> Enable Container-Based Virtual Machines) automatically, too?

Reply:
Good point, I'll follow up. I don't think so, but need to confirm. As far as I understand, it's just that when that setting is checked, CVM is checked when creating a workspace using the UI by defauktl